nitro-node/src/scripts/download-nitro.ts

```diff
@@ -0,0 +1,219 @@
+import fs from "node:fs";
+import path from "node:path";
+import download from "download";
+import { Duplex } from "node:stream";
+
+// Define nitro version to download in env variable ("latest" or tag "v1.2.3")
+const NITRO_VERSION = process.env.NITRO_VERSION || "latest";
+// The platform OS to download nitro for
+const PLATFORM = process.env.npm_config_platform || process.platform;
+// The platform architecture
+//const ARCH = process.env.npm_config_arch || process.arch;
+
+const releaseUrlPrefixVariants = {
+  latest: "https://api.github.com/repos/janhq/nitro/releases/",
```

Comment:
Please leave these configs in rollup.config.js. Some forks may find it difficult to locate and replace here and there.